packages/itwinui-react/src/core/Tree/Tree.tsx

```diff
@@ -57,6 +57,10 @@ type FlatNode<T> = {
 export type NodeRenderProps<T> = Omit<NodeData<T>, 'subNodes'>;
 
 export type TreeProps<T> = {
+  /**
+   * Modify size of the tree.
+   */
+  size?: 'small';
```

Comment:
i think we should also reserve a value for default size

```suggestion
  size?: 'default' | 'small';
```

Reply:
we should probably add a `'default'` string wherever it's missing. it's more explicit that way

a bunch of other components already do it this way
![image](https://github.com/iTwin/iTwinUI/assets/9084735/4ef70847-d75f-46ee-8b99-db2d90a18b4b)